src/lucky_migrator/create_table_statement.cr

```diff
@@ -16,19 +17,32 @@ class LuckyMigrator::CreateTableStatement
     with self yield
     process_rows
     statement << ")"
```

Comment:
The semicolon should go after this

Reply:
K I added the semicolon and I see the error.

After digging around a little, it seems to me that the problem is this `execute` command which is intended for a single sql statement.
https://github.com/luckyframework/migrator/blob/77801d43b2478aafc23d0464bc7cca1e93d4b442/src/lucky_migrator/statement_helpers.cr#L2-L8

One possibility might be to return both the create table statement and the index statements in an array, then execute them individually. But I'm not sure how this would would change with transactions.

I'll look into it tomorrow and let you know.

Reply:
I think that's a great idea actually. Something like:

```
statements = LuckyMigrator::CreateTableStatement.new({{ table_name }}).build do 
   {{ yield }} 
 end.statements # Returns an array of statements like you said

statements.each do |statement|
  execute statement
end
```

I think this will work ok once transactions work 👍 I'm looking forward to seeing how it goes :D

Reply:
That looks good! It's late for me so I'll work on that tomorrow :)

Reply:
Should the build method return the statements directly, or itself?